package.json

```diff
@@ -45,9 +45,13 @@
     "start": "yarn workspace @serlo/frontend start",
     "test": "yarn workspace @serlo/frontend test"
   },
+  "dependencies": {
+    "react-resizable": "^3.0.5"
+  },
   "devDependencies": {
     "@serlo/eslint-config": "workspace:*",
     "@serlo/typescript-config": "workspace:*",
+    "@types/react-resizable": "^3.0.7",
```

Comment:
please remove them here, they should not be in the monorepo root `package.json`

Reply:
Done :heavy_check_mark: